### ISSUE TITLE
"neopixel" is un-selected when moving mouse on it

### ISSUE THREAD

**Issue by Mofell** [Microsoft/pxt#2966]
**Repro steps:**
• Navigate to https://makecode.microbit.org/beta
• Click on Advanced->Add Package
• Click blank area and move(not click) mouse on the three packages

**Expect Result:**
When moving mouse on nepixel, neopixel should be selected with blue border like devices and Bluetooth.
**Actual Result:**
No blue border on neopixel
![addpackage](https://user-images.githubusercontent.com/23447632/30584432-4759d452-9d5c-11e7-90b7-c37b0384f979.gif)
